runtime/gc_base/ObjectAccessBarrier.hpp

```diff
@@ -107,27 +107,20 @@ class MM_ObjectAccessBarrier : public MM_BaseVirtual
 	{
 		MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vmThread);
 
-#if !defined(J9VM_GC_ARRAYLETS)
-		/* non-arraylet */
-		UDATA data = (UDATA)extensions->indexableObjectModel.getDataPointerForContiguous(array);
-		return (void *)(data + (elementSize * (UDATA)index));
-#else /* J9VM_GC_ARRAYLETS */
-#if defined(J9VM_GC_HYBRID_ARRAYLETS)
 		/* hybrid arraylet */
```

Comment:
again, this comment is referring to hybrid arraylet header, which will be on by default now. I'd just remove the comment, it creates confusion with the rest of the code block which deals with array layout (and in fact the very first one to deal with is contiguous one, not hybrid one, which the comment would suggest).